### PR TITLE
Docs/cleanup

### LINKED ISSUE #20: Add license page to doxygen
Add a dedicated \page and add it to the menu of important pages

### PR DESCRIPTION
**Pull request by nekenyu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ It is a general purpose statically typed, easy to learn and use language.
 - strong static typing
 
 #### Manuals:
-- [User Manual](https://jsmith1024.github.io/bill/html/User.html)
-- [Reference Manual](https://jsmith1024.github.io/bill/html/Reference.html)
-- [Bill Primer](https://jsmith1024.github.io/bill/html/Primer.html)
-- [Standard Library Reference](https://jsmith1024.github.io/bill/html/Standard.html)
+- Bill Primer [HTML](https://jsmith1024.github.io/bill/html/Primer.html) [PDF](https://jsmith1024.github.io/bill/Primer.pdf)
+- Reference Manual [HTML](https://jsmith1024.github.io/bill/html/Reference.html) [PDF](https://jsmith1024.github.io/bill/Reference_Manual.pdf)
+- Standard Library Reference [HTML](https://jsmith1024.github.io/bill/html/Standard_Library_Reference.pdf) [PDF](https://jsmith1024.github.io/bill/Library.html)
```
Comment (jsmith1024) [inline README.md]: Addresses are swapped.

File: dox/books/Primer/main_pdf.dox [hunk 2]
```diff
@@ -27,12 +27,10 @@ It is a general purpose statically typed, easy to learn and use language.
 - ease of static typing
 - strong static typing
 
-@htmlonly
 <h4>License</h4>
```
Comment (jsmith1024) [inline dox/books/Primer/main_pdf.dox]: I should have deleted these. In latex, the License notification is in the latex header file.

Please remove the license from this file. (In the reference manual, main_pdf.dox too.)

File: dox/books/Reference_Manual/main_pdf.dox [hunk 3]
```diff
@@ -1,18 +1,12 @@
...
+<h1>Bill Reference Manual</h1>
 <h4>Introduction</h4>
-:
-    - :
-    <ul>
-        <li> 
-    </ul>
+TBD...
 
 <h4>License</h4>
```
Comment (jsmith1024) [inline dox/books/Reference_Manual/main_pdf.dox]: Please remove License notification entirely.

For explanation, please see comment on the Primer's main_pdf.dox.

File: dox/books/Primer/main_pdf.dox (hunk 2 above)
Comment (nekenyu) [reply dox/books/Primer/main_pdf.dox]: Removed from dox/books/*/main_pdf.dox

File: README.md (hunk 1 above)
Comment (nekenyu) [reply README.md]: I believe that was already fixed in another commit.  Note the Outdated next to the filename.

File: doxy [hunk 4]
```diff
@@ -40,7 +38,14 @@ genPdf () {
         popd
         
         # move pdf to docs
-        mv /tmp/bill/${location}/latex/refman.pdf docs/${name}.pdf
+        mv ${tmp_location}/refman.pdf docs/${name}.pdf # TODO: Why is this refman.pdf?
```
Comment (jsmith1024) [inline doxy]: Sounds like a bug.

Comment (jsmith1024) [reply doxy]: Hold on, it generates refman.pdf. I have to auto rename it.

Comment (nekenyu) [reply doxy]: Right.  Just leaving comments for the next person.  Updated comments.

**Review by jsmith1024 — APPROVED** [review]

**Pull request merged**